### DISCUSSION TITLE
Planning for Silicon Salon 3

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#96] [Silicon Salon]
We would like to hold Silicon Salon 3 in either Fall of Winter (depending on how the holidays impact us!).

Please let us know here if you're interested in presenting at Silicon Salon 3, or if you have any specific topic that you'd like to see covered.

(We've considered the topic of accelerators, but that's not set in stone.)

**Comment by kanzure** [comment]
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-November/021213.html

**Reply by ChristopherA** [reply]
Was that just approved?

**Reply by kanzure** [reply]
No, just a backlink. I'm at a Socratic Seminar in Austin.
